.github/workflows/amorphie.core.cache.yml

```
@@ -0,0 +1,27 @@
+jobs:
+  ci-cd-worker-app:
+    if: "!startsWith(github.head_ref, 'feature/')"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Replace NUGET_API_KEY
+        run: |
+          sed -i -e "s/NUGETORG_TOKEN/${{ secrets.NUGETORG_TOKEN }}/g" Amorphie.Core.Cache/Dockerfile
+      - name: Build and push Api
+        uses: docker/build-push-action@v2
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)